rules-reviewed/quarkus/springboot/springboot-data-jpa-to-quarkus.windup.xml

```diff
@@ -16,6 +16,15 @@
     <rules>
         <rule id="springboot-jpa-to-quarkus-00000">
             <when>
+                <!-- Spring component declared in source code condition -->
+                <project>
+                    <artifact groupId="org.springframework.data" artifactId="spring-data-jpa"/>
+                </project>
+                <!-- Spring Boot component declared in source code condition -->
+                <project>
+                    <artifact groupId="org.springframework.boot" artifactId="spring-boot-starter-data-jpa"/>
```

Comment:
here you can see all the artifacts from spring boot that can bring spring-data-jpa : https://mvnrepository.com/artifact/org.springframework.data/spring-data-jpa/usages , just to put an example....this happens with all these dependencies being checked by the project condition

Reply:
Yep, that's true.
The "inferred dependency in source code analysis" use case has been discussed as well and it's not what this rule is trying to cover.
The decision about the "inferred dependency in source code analysis" use case is to not provide any new rule based on the consideration that if it's inferred, it means there are dependant artifacts.
For those dependant artifacts or: 
 - we already have a rule for them that so will fix this as well
 - we have no rule so changing only this depency won't be useful

Reply:
That's why I think checking specific artifacts in the pom only solves a part of the reality and hides another.